src/chatd.cpp

```diff
@@ -3809,7 +3842,21 @@ void Chat::onUserLeave(Id userid)
     if (userid == client().userId())
     {
         mOwnPrivilege = PRIV_NOTPRESENT;
+#ifndef KARERE_DISABLE_WEBRTC
+        mClient.mRtcHandler->onKickedFromChatRoom(mChatId);
+#endif
     }
+#ifndef KARERE_DISABLE_WEBRTC
+    else
+    {
+        uint32_t clientid = 0;
+        if (mClient.mRtcHandler->getClientidFromPeer(mChatId, userid, clientid))
+        {
+            onEndCall(userid, clientid);
```

Comment:
It's a double check, in case the client of the user, that has been removed from chat room, doesn't finish its call participation, we finish our session with it. With our code, this case isn't possible but with some client modification would be possible